<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManager</class>
 <widget class="QDialog" name="UserManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../SysMainWindow.qrc">
    <normaloff>:/SysMainWindow/Resources/login-icon.png</normaloff>:/SysMainWindow/Resources/login-icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>权限</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>68</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_BianMaJianShiCaoZuo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>编码检视操作</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox_BianMaJianShiJianDu">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>编码检视监督</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_XinXiGuanLiCaoZuo">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>信息管理操作</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox_XinXiGuanLiJianDu">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>信息管理监督</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_27">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(41, 136, 41);</string>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_XingMing">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_29">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(41, 136, 41);</string>
       </property>
       <property name="text">
        <string>军官证号</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_JunGuanZheng">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_JiuMiMa">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(41, 136, 41);</string>
       </property>
       <property name="text">
        <string>旧密码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_JiuMiMa">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_26">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(41, 136, 41);</string>
       </property>
       <property name="text">
        <string>密码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_MiMa">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_28">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(41, 136, 41);</string>
       </property>
       <property name="text">
        <string>确认密码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_QueRenMiMa">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox_XianShi">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>显示密码</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox_XiuGaiMiMa">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>修改密码</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_QueDing">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;微软雅黑&quot;;
color: rgb(41, 136, 41);</string>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_BianMaJianShiCaoZuo</tabstop>
  <tabstop>checkBox_BianMaJianShiJianDu</tabstop>
  <tabstop>checkBox_XinXiGuanLiCaoZuo</tabstop>
  <tabstop>checkBox_XinXiGuanLiJianDu</tabstop>
  <tabstop>lineEdit_XingMing</tabstop>
  <tabstop>lineEdit_JunGuanZheng</tabstop>
  <tabstop>lineEdit_JiuMiMa</tabstop>
  <tabstop>lineEdit_MiMa</tabstop>
  <tabstop>lineEdit_QueRenMiMa</tabstop>
  <tabstop>checkBox_XianShi</tabstop>
  <tabstop>checkBox_XiuGaiMiMa</tabstop>
  <tabstop>pushButton_QueDing</tabstop>
 </tabstops>
 <resources>
  <include location="../../SysMainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox_XiuGaiMiMa</sender>
   <signal>clicked(bool)</signal>
   <receiver>UserManager</receiver>
   <slot>SetModifyPasswordChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_QueDing</sender>
   <signal>clicked()</signal>
   <receiver>UserManager</receiver>
   <slot>ConfirmModification()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_XianShi</sender>
   <signal>clicked(bool)</signal>
   <receiver>UserManager</receiver>
   <slot>ShowPassword(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetModifyPasswordChecked(bool)</slot>
  <slot>ConfirmModification()</slot>
  <slot>ShowPassword(bool)</slot>
 </slots>
</ui>
